pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -10,12 +9,16 @@ import { debounce } from 'lodash';
 import { useLabeledFormElement, labeledFormElementProps } from '@shell/composables/useLabeledFormElement';
 import { useCompactInput } from '@shell/composables/useCompactInput';
 
-declare module 'vue/types/vue' {
-  interface Vue {
-    onInput: (event: Event) => void | ((event: Event) => void);
-  }
+interface NonReactiveProps {
+  onInput: (event: Event) => void | ((event: Event) => void);
 }
 
+const provideProps: NonReactiveProps = {
```

Comment:
I apologize for asking after 4 E2E tests fail retry and the late question.
May I ask you why do we need these noops for the setup?

Reply:
The `noop` comment is a strategy to resolve the following tsc error and communicate that we expect this function to be empty at this point:

```
Unexpected empty method 'onInput'.eslint[@typescript-eslint/no-empty-function](https://github.com/typescript-eslint/typescript-eslint/blob/v5.4.0/packages/eslint-plugin/docs/rules/no-empty-function.md)
```

An alternative approach would be add `// eslint-disable-next-line @typescript-eslint/no-empty-function` above the offending function.

Reply:
We could disable the TS lint rule for `*.vue` files at this point and keep it undefined.

Reply:
It makes sense to consider, although such a change would be out of scope for this PR.

Reply:
Totally agree, which is why I did not ask it 😄